packages/outputarea/src/widget.ts

```diff
@@ -400,7 +400,21 @@ class OutputArea extends Widget {
       if (isolated === true) {
         output = new Private.IsolatedRenderer(output);
       }
-      output.renderModel(model);
+      output.renderModel(model).catch(error => {
+         // Add stderr message to output
+        const stderr = `Javascript Error: ${error.message}. This usually means there's a typo in your chart specification. See the JavaScript console for the full traceback.`;
```

Comment:
This error message is not particulary generic, since output types don't necessarily have a chart spec.

Reply:
Good catch. That's leftover from vega3-extension-specific error message 😬

Reply:
This approach is pretty hacky BTW. I tried creating a new widget using `this.rendermime.createRenderer` with the `application/vnd.jupyter.stderr` mime type and replacing the current widget with it (which would be much less hacky) but it doesn't work because `return widget` below happens before this promise ever catches.